docs/certificates.md

```diff
@@ -0,0 +1,171 @@
+To use the custom certificate created using the steps from [*GENERATING A NEW CERTIFICATE*](#generating-a-new-certificate):
+* copy the certificates into `/etc/pki/tls/certs/` directory:
+> :warning: make sure to not modify the destination file names and path (`/etc/pki/tls/certs/pulp_webserver.crt` and `/etc/pki/tls/private/pulp_webserver.key`) because these are the names configured in Nginx
+```console
+$ podman exec -it pulp bash
+[root@f14649b06e01 /]# cp /etc/pulp/certs/pulp_webserver.crt /etc/pki/tls/certs/pulp_webserver.crt
+cp: overwrite '/etc/pki/tls/certs/pulp_webserver.crt'? y
+[root@f14649b06e01 /]# cp /etc/pulp/certs/pulp_webserver.key /etc/pki/tls/private/pulp_webserver.key
+cp: overwrite '/etc/pki/tls/private/pulp_webserver.key'? y
+```
+
+* restart nginx process to get the new certificate
+```
+$ podman exec pulp s6-svc -r /run/service/nginx
+```
+
+* verify that Nginx is now using the new certificate
+```
+$ podman exec pulp  openssl s_client -connect pulp:443
+Can't use SSL_get_servername
+depth=0 CN = pulp.example.com
+verify error:num=18:self signed certificate
+verify return:1
+depth=0 CN = pulp.example.com
+verify return:1
+CONNECTED(00000003)
+---
+Certificate chain
+ 0 s:CN = pulp.example.com
+   i:CN = pulp.example.com
+---
+Server certificate
+-----BEGIN CERTIFICATE-----
+MIIDTTCCAjWgAwIBAgIUZFS+5+hhWrM270+X+k8vpfwIQL8wDQYJKoZIhvcNAQEL
+BQAwGzEZMBcGA1UEAwwQcHVscC5leGFtcGxlLmNvbTAeFw0yMjEyMTQxNzMxMzda
+Fw0yMzEyMTQxNzMxMzdaMBsxGTAXBgNVBAMMEHB1bHAuZXhhbXBsZS5jb20wggEi
+...
+-----END CERTIFICATE-----
+subject=CN = pulp.example.com
+issuer=CN = pulp.example.com
+```
+
+### **Configuring an existent certificate**
+
+It is also possible to bring your own company certificate instead of creating a new one through [*GENERATING A NEW CERTIFICATE*](#generating-a-new-certificate) steps.
+
+* copy the certificates into `/etc/pulp/certs/` directory. This will prevent having to copy the certificates again in case of a container reprovisioning (the certificates will be persisted in container volume):
+```console
+$ podman cp my-company-generated-certificate.crt pulp:/etc/pulp/certs/pulp_webserver.crt
+$ podman cp my-company-generated-certificate.key pulp:/etc/pulp/certs/pulp_webserver.key
+```
+
+* now, copy the certificates into `/etc/pki/tls/certs/` directory:
+> :warning: make sure to not modify the destination file names and path (`/etc/pki/tls/certs/pulp_webserver.crt` and `/etc/pki/tls/private/pulp_webserver.key`) because these are the names configured in Nginx
+```console
+$ podman exec -it pulp bash
+[root@f14649b06e01 /]# cp /etc/pulp/certs/pulp_webserver.crt /etc/pki/tls/certs/pulp_webserver.crt
+cp: overwrite '/etc/pki/tls/certs/pulp_webserver.crt'? y
+[root@f14649b06e01 /]# cp /etc/pulp/certs/pulp_webserver.key /etc/pki/tls/private/pulp_webserver.key
+cp: overwrite '/etc/pki/tls/private/pulp_webserver.key'? y
+```
+
+* restart nginx process to get the new certificate
+```
+$ podman exec pulp s6-svc -r /run/service/nginx
+```
```

Comment:
This process is not sufficient. We have  additional commands to run also:
https://github.com/pulp/pulp-oci-images/blob/latest/images/s6_assets/init/certs#L38

Reply:
hum... the addition of the CA into the truststore (which is done by `update-ca-trust` command) is documented in the next section (## SETTING UP ADDITIONAL TRUSTED CAs).
https://github.com/pulp/pulp-oci-images/pull/397/files#diff-b29b72c39bf9c8efd205b812ea4ada8c1a7598b4a04bb632b3345ad39155f462R150
https://github.com/pulp/pulp-oci-images/pull/397/files#diff-b29b72c39bf9c8efd205b812ea4ada8c1a7598b4a04bb632b3345ad39155f462R169


Considering that users could be configuring a certificate signed by a trusted CA (like certsign, letsencrypt, etc) I believe that https://github.com/pulp/pulp-oci-images/blob/latest/images/s6_assets/init/certs#L38 is not a mandatory step, right?


Another thing to notice is that "force-enable" and/or "extract" is/are not needed anymore:
```
$ cat /usr/bin/update-ca-trust
#!/usr/bin/sh

#set -vx

# At this time, while this script is trivial, we ignore any parameters given.
# However, for backwards compatibility reasons, future versions of this script must 
# support the syntax "update-ca-trust extract" trigger the generation of output 
# files in $DEST.
...
```